learn/configuration/settings.md

```diff
@@ -234,7 +234,7 @@ Suppose you would like to search `the mask` in a movie database. Since `the` is
 
 ## Synonyms
 
-A set of words defined for an index. Synonyms are **different words that have the same meaning**, and thus are treated similarly. If either of the associated words is searched, the same results will be displayed.
+A set of words defined for an index. Synonyms are **different words that have the same meaning**, and thus are treated similarly.
```

Comment:
I wonder if "treated" by itself is clear enough? I find it a bit vague without the deleted sentence, as if treated might mean loads of different and mysterious things 🤔

Reply:
You're right. I got rid of the sentence because `the same results will be displayed` isn't always the case. I updated it to `get some of the same results`.